content/en/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md

```diff
@@ -0,0 +1,264 @@
+
+2. On the **Basic Information** page, name the workspace as `devops-multicluster`, select `ws-admin` for **Administrator**, and click **Next**.
+
+   ![create-workspace](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/create-workspace.png)
+
+3. On the **Select Clusters** page, select all three clusters and click **Create**.
+
+   ![select-all-clusters](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/select-all-clusters.png)
+
+4. The workspace created will display in the list. You need to log out of the console and log back in as `ws-admin` to invite both `project-admin` and `project-regular` to the workspace and grant them the role `workspace-self-provisioner` and `workspace-viewer` respectively. For more information, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/#step-2-create-a-workspace).
+
+   ![workspace-created](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/workspace-created.png)
+
+### Step 3: Create a DevOps project
+
+1. Log out of the console and log back in as `project-admin`. Go to the **DevOps Projects** page and click **Create**.
+
+2. In the dialog that appears, enter `multicluster-demo` for **Name**, select **host** for **Cluster Settings**, and then click **OK**.
+
+   ![devops-project](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/devops-project.png)
+
+   {{< notice note >}}
+
+   Only clusters with the DevOps component enabled will be available in the drop-down list.
+
+   {{</ notice >}}
+
+3. The DevOps project created will display in the list. Make sure you invite the account `project-regular` to this project with the role `operator`. For more information, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/#step-5-create-a-devops-project-optional).
+
+   ![devops-project-created](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/devops-project-created.png)
+
+### Step 4: Create projects on clusters
+
+You must create the projects as shown in the table below in advance. Make sure you invite the account `project-regular` to these projects with the role `operator`. For more information about how to create a project, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/#step-3-create-a-project).
+
+| Cluster Name | Description             | Project Name           |
+| ------------ | ----------------------- | ---------------------- |
+| host         | Cluster for testing     | kubesphere-sample-prod |
+| shire        | Cluster for production  | kubesphere-sample-prod |
+| rohan        | Cluster for development | kubesphere-sample-dev  |
+
+### Step 5: Create credentials
+
+1. Log out of the console and log back in as `project-regular`. On the **DevOps Projects** page, click the DevOps project **multicluster-demo**.
```

Comment:
the DevOps project **multicluster-demo**.
->
the DevOps project `multicluster-demo`.